internal/metarepos/report_collector.go

```diff
@@ -681,14 +680,10 @@ func (rce *reportCollectExecutor) processReport(response *snpb.GetReportResponse
 			i++
 		} else {
 			if cur.Version < prev.Version {
-				fmt.Printf("invalid report prev:%v, cur:%v\n",
-					prev.Version, cur.Version)
-				rce.logger.Panic("invalid report",
+				rce.logger.Error("invalid report",
```

Comment:
Could you please review this? This PR introduces a change, but it's not covered by the added test, TestReportHandleInvalidReport.

Reply:
omg. Invalid report is filtered by other conditions. I fixed it.